server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -74,28 +74,50 @@ public class ThreadPool implements ReportingService<ThreadPoolInfo>, Scheduler,
      */
```

Comment:
In the comments below on the particular thread pools, you mention thread pool sizes, but those are defined elsewhere -- it looks like in `DefaultBuiltInExecutorBuilders#getBuilders`? It seems helpful to mention that class in this comment, maybe? 

Not ideal/modular to reference another class like that, but otherwise it's a bit difficult to track down the context.

Oh actually, it looks like the code was recently moved out of this file (see https://github.com/elastic/elasticsearch/commit/ae41e9ab65ec4b662f21810e16292742897ee674), that's why I had a hard time. The call to `getBuilders` in `ThreadPool#ThreadPool` is undocumented, as well, if you care to improve that.

Reply:
Ping

please mention DefaultBuiltInExecutorBuilders#getBuilders builds the thread pools -- took me time to find where the code went, and it's relevant to the new documentation.

Reply:
:+1: done